app/templates/pdf/speakers.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>{{ event.name }} - {{ ("Speakers List") }}</title>
+    <style type="text/css">
+        {% include 'bootstrap/bootstrap_pdf.css' %}
+        table th {
+            text-align: left;
+        }
+
+        table td {
+            text-align: left;
+            border-spacing: 1em;
+        }
+
+        table {
+            margin-top: 20px;
+            border-collapse: separate;
+        }
+
+        .total-row {
+            margin-top: 25px;
+        }
+
+        .speaker-total {
+            margin-right: 50px;
+        }
+
+        tr.padded td{
+        padding-top : 40px;
+        }
+    </style>
+</head>
+<body id="body">
+<h3 style="text-align:center;">{{ event.name }} - {{ ("Speakers List") }}</h3>
+    <br>
+    <table class="table with-datatable" id="speakers-table" width="100%">
+        <thead>
+        <tr>
+            <th>
+                {{ ("Speaker") }}
+            </th>
+            <th>
+                {{ ("Speaker Details") }}
+            </th>
+            <th>
+                {{ ("User") }}
+            </th>
+            <th>
+                {{ ("User Details") }}
+            </th>
+        </tr>
+        </thead>
+        <tbody>
+        {% for speaker in speakers %}
+            {% if speaker.status != "deleted" %}
+                <tr class="padded">
+                    <td><br>
+                            Speaker Number    :<br>
+                            Speaker By        :<br>
+                            Speaker Status    :<br>
+                            Quantity        :<br>
+                            Total Amount    :<br>
+                            Payment Mode    :<br>
+                            Discount Code   :<br>
+                        {% if speaker.status == 'completed' %}
+                            Speaker Completed :<br>
+                        {% else %}
+                            Speaker Placed    :<br>
+                        {% endif %}
+                    </td>
+                    <td><br>
+                            <strong>{{ speaker.get_invoice_number() }}</strong><br>

```

Comment:
Speaker doesn't have an invoice number :( Just check the speaker.py file in models. Look at exporting speakers as csv. You need to print all those fields.

Reply:
exporting speakers as csv is similiar to exporting orders as csv. where to get all the fields?

Reply:
Have a look at #5156

Reply:
All the models are located in `app/models`

Reply:
I didnt understand how to get all the fields and which all fields to put